Add copy files phase to copy the static library header files

Unrelentless opened the pull request:
Add copy files phase to copy the headers as per:

[Using Static Libraries in iOS#Configuring Exported Headers](https://developer.apple.com/library/ios/technotes/iOSStaticLibraries/Articles/creating.html#//apple_ref/doc/uid/TP40012554-CH2-SW4)

Commit:
Add copy files phase to copy the static library headers.

https://developer.apple.com/library/ios/technotes/iOSStaticLibraries/Articles/creating.html#//apple_ref/doc/uid/TP40012554-CH2-SW4

sharplet commented on FBTweak.xcodeproj/project.pbxproj:
```diff
@@ -347,6 +381,7 @@
 				18EFE46A189EBA4900DA6A5D /* Sources */,
 				18EFE46B189EBA4900DA6A5D /* Frameworks */,
 				18EFE497189EBA5A00DA6A5D /* Headers */,
```
You should also delete the existing Headers phase.

Commit:
Remove `$(SYMROOT)/Headers` from header search paths in example project

It is no longer needed with the custom copy headers phase as cited in:
https://developer.apple.com/library/ios/technotes/iOSStaticLibraries/Articles/creating.html#//apple_ref/doc/uid/TP40012554-CH2-SW4

> The include folder inside a build products directory is in the default header search path for applications, so this is an appropriate place to put header files.

sharplet commented:
Given that this is fixing a broken `archive` build action, I think we should add a step to the `.travis.yml` file that tests the archive action is working correctly.

I think you'll need to share the `FBTweakExample` scheme for this to work from the command line on Travis.

Commit:
Update .travis.yml to test archive action works correctly

sharplet commented:
Thanks @Unrelentless! I've pulled your commits and opened https://github.com/facebook/Tweaks/pull/100.
